test/parallel/test-transform-by.js

```diff
@@ -0,0 +1,286 @@
+'use strict';
```

Comment:
Added by mistake perhaps?

Reply:
This is not an ESM, so this is needed.

Reply:
@mcollina I meant the file itself. =) Looks like @ronag already removed it.

Reply:
My mistake. Sorry about that.